go/server/handlers.go

```diff
@@ -33,8 +34,22 @@ func (s *Server) informationHandler(c *gin.Context) {
 }
 
 func (s *Server) homeHanlder(c *gin.Context) {
```

Comment:
nit: `homeHanlder` -> `homeHandler`

Reply:
Fixed in d230e96ed7486779a71b7f916a9784bfca01b336